install/aws-installation-guide.md

```diff
@@ -104,7 +104,7 @@ This command will produce output similar to the following:
 ]
 ```
 
-Launch the EC2 instance and pick ip the `INSTANCEID`:
+Launch the EC2 instance and pick IP the `INSTANCEID`:
```

Comment:
This doesn't read right. Maybe it was mean to say 'pick up the `INSTANCEID`' ?

Reply:
@raravena80 - can you comment on this?

Reply:
We can always reword in a follow-up PR but this is just a "literal tidyup".

Reply:
i think up makes more sense here.